cmake/external-libraries.cmake

```diff
@@ -41,6 +41,20 @@ if(KPHP_TESTS)
     endif()
 endif()
 
+find_library(KPHP_TIMELIB kphp-timelib)
+if(KPHP_TIMELIB)
+    add_library(kphp-timelib STATIC IMPORTED ${KPHP_TIMELIB})
+else()
+    handle_missing_library("kphp-timelib")
+    FetchContent_Declare(kphp-timelib GIT_REPOSITORY https://github.com/VKCOM/timelib)
+    message(STATUS "---------------------")
+    FetchContent_MakeAvailable(kphp-timelib)
+    include_directories(${kphp-timelib_SOURCE_DIR}/include)
+    target_compile_options(kphp-timelib PUBLIC -Wno-float-conversion -std=gnu11)
```

Comment:
Do we need this? Can we put this options directly to kphp-timelib CMakeList.txt?

Reply:
I moved these flags to kphp-timelib repository and it seems to be working.